src/Dfe.Spi.GraphQlApi.Domain/Configuration/EntityRepositoryConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+namespace Dfe.Spi.GraphQlApi.Domain.Configuration
+{
+    public class EntityRepositoryConfiguration
+    {
+        public string SquasherApiBaseUrl { get; set; }
+        public string SquasherApiFunctionKey { get; set; }
```

Comment:
Probably doesn't have to be done now (i.e. just to get the demo going), but might be a good idea to refer to generic "headers".. that way, when we move to Facades instead of going direct to function app, we'll have a subscription key instead of an API function key, so means you'll be able to rename the name of the header as well as the value.

So maybe in your settings just supply key/value pairs, and here have a `Dictionary<string, string>`? Just a suggestion.

Reply:
I tend to use well-defined names. So SquasherSubscriptionKey, SquasherClientId. And then add the relevant headers, authenticators as appropriate. We can look at this generically for all when we do the internal facades though.